x-pack/test/saved_object_api_integration/common/suites/export.ts

```diff
@@ -155,8 +155,16 @@ export function exportTestSuiteFactory(esArchiver: any, supertest: SuperTest<any
     if (failure?.reason === 'unauthorized') {
       // In export only, the API uses "bulkGet" or "find" depending on the parameters it receives.
       if (failure.statusCode === 403) {
-        // "bulkGet" was unauthorized, which returns a forbidden error
-        await expectSavedObjectForbiddenBulkGet(type)(response);
+        if (id) {
+          // "bulkGet" was unauthorized, which returns a forbidden error
+          await expectSavedObjectForbiddenBulkGet(type)(response);
+        } else {
+          expect(response.body).to.eql({
+            statusCode: 403,
+            error: 'Forbidden',
+            message: `unauthorized`,
+          });
+        }
```

Comment:
Previously, per-id unauthorized were returning a 403 (due to bulkGet returning an error) and per-type were returning an empty export instead.

This PR has the side effect to change that, to now returns a 403 for per-type export when the user don't have the permissions for any of the types. 

Overall, I think this is an improvement in term of consistency.